ports/psoc6/Makefile

```diff
@@ -43,35 +48,44 @@ MPY_MAIN_BUILD_DIR := $(dir $(MPY_DIR_OF_MAIN_MAKEFILE))/$(BUILD)
 $(info MPY_PATH_TO_MAIN_MAKEFILE        : $(MPY_PATH_TO_MAIN_MAKEFILE))
 $(info MPY_DIR_OF_MAIN_MAKEFILE         : $(MPY_DIR_OF_MAIN_MAKEFILE))
 $(info MPY_MAIN_BUILD_DIR               : $(MPY_MAIN_BUILD_DIR))
+$(info CONFIG                           : $(CONFIG))
 
 
 INC += -I.
 INC += -I$(TOP)
 INC += -I$(BUILD)
+INC += -I$(TOP)/lib/mbedtls/include
```

Comment:
This seems to be more relevant to the extmod.mk file --> https://github.com/jaenrig-ifx/micropython/blob/ports/psoc6/extmod/extmod.mk#L132-L212
Any reason to make such inclusion here?

Reply:
I have not found out yet why the extmod.mk is not sufficient, but this needed to be added in order to compile properly. In general I agree, this flags in extmod.mk should suffice. Something to be investigated.

Reply:
Already found it -> https://github.com/jaenrig-ifx/micropython/blob/78345c36baa8a747da3a138152eb6e58231b7df4/ports/psoc6/Makefile#L71
In extmod these varibles are already defined,
CFLAGS should have an append assignment (+=) as in extmod.mk (maybe other includes as well) they are already declaring some flags.
Same goes for the empty LIBS assignment. Could be the case for other variables as well, but that I have not investigated.

Reply:
Fixed now also for other flags